includes/Services/SiteGenService.php

```diff
@@ -125,12 +125,48 @@ public static function instantiate_site_meta( $site_info, $identifier, $skip_cac
 		return $response;
 	}
 
+	/**
+	 * Generate child them slug and name based on the generated title.
+	 *
+	 * @param array $site_config The site config generated by AI.
+	 * @param array $data The homepage related data.
+	 * @param array $homepage_index The index of the theme version.
+	 * @return array
+	 */
+	public static function update_info_for_child_theme( $site_config, $data, $homepage_index = '' ) {
+		// Fetch title and slug for the child theme
+		$theme_name = \remove_accents( $site_config['site_title'] );
+		$theme_slug = \sanitize_title_with_dashes( $theme_name );
+
+		$homepage_slug              = $data['slug'];
+		$title                      = str_replace( ' ', '-', str_replace( array( '(', ')' ), '', $data['title'] ) );
+		$homepage_title_with_dashes = strtolower( str_replace( ' ', '-', $title ) );
```

Comment:
Doesn't the $title already replace '-' with ' '?

Reply:
fixed ... removed this variable $homepage_title_with_dashes and using $title itself